packages/front-end/components/Experiment/ResultMoreMenu.tsx

```diff
@@ -97,7 +92,9 @@ export default function ResultMoreMenu({
             <BsArrowRepeat className="mr-2" /> Re-run All Queries
           </button>
         )}
-      {hasData && queries && generateReport && canEdit && snapshotId ? (
```

Comment:
question: Now that we removed the `hasData` & `queries` from here, does that change the behavior of when this button shows up?

I am curious if this is related to the comment we have in `reports.tsx`:

```
<em>Note:</em> you will not see the &quot;New Custom Report&quot;
          option if your experiment does not have results yet or is not hooked
          up to a valid data source.
```